homeassistant/base/Dockerfile

```diff
@@ -133,3 +133,13 @@ RUN apk add --no-cache \
     && apk del \
         mariadb-dev postgresql-dev freetds-dev \
         gcc g++ musl-dev python3-dev make
+
+####
+## Install gatttool from edge (can be removed with release of Alpine 3.7)
+RUN mkdir -p bluez-deprecated \
+    && cd bluez-deprecated \
+    && curl -o bluez-deprecated-5.47-r3.apk http://dl-3.alpinelinux.org/alpine/edge/main/armhf/bluez-deprecated-5.47-r3.apk \
```

Comment:
You need write that for all arch. You can use `$BUILD_ARCH` they include (amd64, aarch64, armhf, i386)